content/asset-tracking/index.textile

```diff
@@ -0,0 +1,163 @@
+- @minimumDisplacement@ := Minimum positional granularity required, in metres. Lowering this value increases the spatial resolution. Location updates whose position differs from the last known position by a distance smaller than this value are to be filtered out. Used to configure the underlying location provider, as well as to filter the broadcast of updates to subscribers.
+
+h3(#request-different-resolution). Request a different Resolution 
+
+The subscriber may require different resolutions of updates depending on state. For example, whether the map is currently displayed to the user or not, what zoom level the map is set to, and so on. This feature allows the subscriber to specify different preferences depending on any factors of interest to the client.
+
+h2(#resolution-set). Resolution set
+
+A resolution set, is a set of resolutions that must be provided by the app in order to inform the resolution policy. While developers can define their own resolution sets, Ably also provides a default resolution set that can be used with the default resolution policy.
+
+h3(#default-resolution-set). Default resolution set
+
+The @DefaultResolutionSet@ is the set of resolutions which must be defined in order to satisfy the "default resolution constraints":#default-resolution-constraints. These are required in order to inform the "default resolution policy":#default-resolution-policy.
+
+For the @DefaultResolutionSet@ the following resolutions are expected to be included:
+
+- @farWithoutSubscriber@ := The resolution to select if above the "@proximityThreshold@":#default-resolution-constraints, with no subscribers.
+- @farWithSubscriber@ := The resolution to select if above the "@proximityThreshold@":#default-resolution-constraints, with one or more subscribers.
+- @nearWithoutSubscriber@ := The resolution to select if below the "@proximityThreshold@":#default-resolution-constraints, with no subscribers.
+- @nearWithSubscriber@ := The resolution to select if below the "@proximityThreshold@":#default-resolution-constraints, with one or more subscribers.
+
+h2(#resolution-constraints). Resolution constraints
+
+Resolution constraint objects, @ResolutionConstraints@, are defined on a per-@Trackable@ basis, and are then used to calculate an appropriate @Resolution@ by the @ResolutionPolicy@.
+
+h3(#default-resolution-constraints). Default resolution constraints
+
+Ably provides default resolution constraints consisting of:
+
+- @DefaultResolutionSet@ := The set of resolutions which must be defined in order to populate the @DefaultResolutionConstraints@.
+- @DefaultProximity@ := This is the proximity to the destination at which resolution of updates is increased to improve user experience. Proximity can be spatial, temporal, or both. 
+- @batteryLevelThreshold@ := This is a level of battery on the device that runs Publishing SDK that is considered low. When the battery level falls below this, the resolution of updates is reduced.
```

Comment:
"This is a level of battery on the device that runs Publishing SDK that is considered low" -> "This is a battery level for the publisher's device that is considered to be low"

Reply:
Much better, thanks.